<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nodePopupClass</class>
 <widget class="QMainWindow" name="nodePopupClass">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Node Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="nodePopupGrid">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="1" column="1">
     <widget class="QGroupBox" name="nodeGroupBox">
      <property name="title">
       <string>Node Settings</string>
      </property>
      <layout class="QFormLayout" name="nodeFormBox">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="ipLabel">
         <property name="toolTip">
          <string>This is the IP Address the node will attempt to use when using static IP addressing.</string>
         </property>
         <property name="text">
          <string>IP Address</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="ipHBox">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="ipSpinBox0">
           <property name="wrapping">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="ipSpinBox1">
           <property name="wrapping">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="ipSpinBox2">
           <property name="wrapping">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="ipSpinBox3">
           <property name="wrapping">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="ethernetLabel">
         <property name="toolTip">
          <string>The prefix of the ethernet address is determined by the ethernet vendor, to change this either edit the node's profile or create a custom profile for the node.</string>
         </property>
         <property name="text">
          <string>Ethernet Vendor</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="ethernetVendorEdit">
         <property name="toolTip">
          <string>The prefix of the ethernet address is determined by the ethernet vendor, to change this either edit the node's profile or create a custom profile for the node.</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="macLabel">
         <property name="toolTip">
          <string>This is the MAC address the node will use when replying to arp requests or attempting to get a DHCP lease on an IP Address.</string>
         </property>
         <property name="text">
          <string>Ethernet Address</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="ethernetHBox">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="generateButton">
           <property name="toolTip">
            <string>Generates a MAC address currently unused by the Haystack using the Ethernet Vendor to determine the prefix.</string>
           </property>
           <property name="text">
            <string>Generate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QFrame" name="bBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="buttonHBox">
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="bBoxHSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="restoreButton">
         <property name="text">
          <string>Restore</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="bBoxHSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="applyButton">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="bBoxHSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="groupLHSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="groupVSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <spacer name="groupRHSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
